### DISCUSSION TITLE
Splash screen scale

### DISCUSSION THREAD

**Discussion by aktivdigital-frontend** [ionic-team/capacitor#6516] [Q&A]
Hi!

I tried every solution on internet, but splash screen always distorted...

In my capacitor.config.json:

"plugins": {
		"SplashScreen": {
			"launchAutoHide": false,
			"androidScaleType": "CENTER_CROP",
			"showSpinner": true,
			"splashFullScreen": false,
			"splashImmersive": false
		}
	}

How can I solve this? Why dont work CENTER_CROP and other options? I use Capacitor 4.

Thank you!

**Comment by angeloraso** [comment]
Hi [aktivdigital-frontend](https://github.com/aktivdigital-frontend)
The splash looks stretched for the sides. I solve this problem by creating a "stretched" splash (2120x1920). I hope it helps you
